.github/workflows/integration-linux.yml

```diff
@@ -0,0 +1,76 @@
+name: Nightly Linux integration tests
+
+on:
+  schedule:
+  - cron: "0 10 * * *"
+
+jobs:
+
+  build:
+    name: Release binary integration tests
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.14
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install Kustomize
+      run: |
+        wget -O kustomize.tar.gz https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v3.5.4/kustomize_v3.5.4_linux_amd64.tar.gz
+        sudo tar -xvf kustomize.tar.gz -C /usr/local/bin/
+
+    - name: Install Ko
+      run: |
+        wget -O ko.tar.gz https://github.com/google/ko/releases/download/v0.4.0/ko_0.4.0_Linux_x86_64.tar.gz
+        sudo tar -xvf ko.tar.gz -C /usr/local/bin/
+
+    - name: Install Kompose
+      run: |
+        wget -O kompose https://github.com/kubernetes/kompose/releases/download/v1.21.0/kompose-linux-amd64 && chmod +x kompose
+        sudo mv kompose /usr/local/bin/
+
+    - name: Install GCloud
+      run: |
+        wget -O gcloud.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-276.0.0-linux-x86_64.tar.gz
+        sudo tar -xvf gcloud.tar.gz -C /
+        sudo CLOUDSDK_PYTHON="python2.7" /google-cloud-sdk/install.sh --usage-reporting=false --bash-completion=false --disable-installation-options
+        echo "::add-path::/google-cloud-sdk/bin"
+
+    - name: Configure GCloud with Docker
+      run:  sudo gcloud auth configure-docker
```

Comment:
Q: Does this do more than configure docker-credential-gcloud?  Does it need credentials?

Reply:
> I'm new to Github Actions so excuse naïve questions. Can we create reuseable scripts with pluggable values? Wondering if we can create parallel tests to verify compatibility with our minimum supported versions.

I also just started learning about Github Actions last week and figuring things as I go along. I can try to template out some of these versions, let me see if I can find examples in their documentation.

Reply:
> Q: Does this do more than configure docker-credential-gcloud? Does it need credentials?

I based the steps 1:1 against the way we build the existing integration test [Dockerfile](https://github.com/GoogleContainerTools/skaffold/blob/master/deploy/skaffold/Dockerfile.deps). I think this step might be redundant, I'll try testing without it.

Reply:
> Q: Does this do more than configure docker-credential-gcloud? Does it need credentials?

To actually answer the question: It doesn't need any creds and I think it only populates the docker `config.json`:

```
{
  "credHelpers": {
    "us.gcr.io": "gcloud",
    "asia.gcr.io": "gcloud",
    "marketplace.gcr.io": "gcloud",
    "gcr.io": "gcloud",
    "eu.gcr.io": "gcloud",
    "staging-k8s.gcr.io": "gcloud"
  }
}
```
Also I subsequently clear out this file:
`echo '{}' > /home/$(whoami)/.docker/config.json` so I don't think that this step is really needed.

Reply:
Tried without the line `sudo gcloud auth configure-docker` and it resulted in a few failures. There's probably additional configuration that this does. I'll leave it as is 😄